### PR TITLE
chore: align confirmsignmessage to use navstate

### PR DESCRIPTION
**Pull request by im-adithya**
### Describe the changes you have made in this PR

Uses navstate in confirmsignmessage

### Link this PR to an issue

#1457

### Type of change (Remove other not matching type)

- `feat`: New feature (non-breaking change which adds functionality)

### Checklist

- [x] My code follows the style guidelines of this project and performed a self-review of my own code
- [x] New and existing tests pass locally with my changes
- [x] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)

### CODE REVIEW

File: src/app/screens/Keysend/index.tsx [hunk 1]
```diff
@@ -7,29 +7,20 @@ import SatButtons from "@components/SatButtons";
 import SuccessMessage from "@components/SuccessMessage";
```
Comment (escapedcat) [inline src/app/screens/Keysend/index.tsx:7]: This shouldn't be in this PR, right?
It's covered in #1488 I think

Comment (im-adithya) [reply src/app/screens/Keysend/index.tsx]: I didn't checkout from master it seems :P

File: src/app/screens/ConfirmSignMessage/index.tsx [hunk 2]
```diff
@@ -8,25 +8,20 @@ import { useState, useRef } from "react";
...
+function ConfirmSignMessage() {
+  const navState = useNavigationState();
   const { t: tCommon } = useTranslation("common");
   const { t } = useTranslation("translation", {
     keyPrefix: "confirm_sign_message",
   });
 
-  const messageRef = useRef(props.message);
-  const originRef = useRef(props.origin || getOriginData());
+  const messageRef = useRef(navState.args?.message as string);
```
Comment (escapedcat) [inline src/app/screens/ConfirmSignMessage/index.tsx:23]: Ah, same questions here regarding if we need `useRef` here.

**Review by escapedcat — APPROVED** [review]
ack

**Pull request merged**